bittensor/_neuron/text/core_server/run.py

```diff
@@ -442,17 +442,18 @@ def backward_callback(inputs_x:torch.FloatTensor, grads_dy:torch.FloatTensor, sy
                 # --- query the chain for the most current number of peers on the network
                 chain_weights = torch.zeros(subtensor.n)
                 chain_weights [ uid ] = 1 
```

Comment:
Should also encapsulate these lines and `try` should surround just the inner bit of the new `if`